### ISSUE TITLE
a11y: donor stats in donor profile page

### ISSUE THREAD

**Issue by ramiy** [WordImpress/Give#931]
The donor stats are using `<ul>`, why? it's not a list.

Should be replaced with `<div>` or `<p>`.

![donor-stats](https://cloud.githubusercontent.com/assets/576623/17852049/d5ee0348-686d-11e6-9ba2-e960af12b458.png)

**Comment by DevinWalker** [comment]
Is it causing an issue? It's an inline list.

**Comment by ramiy** [comment]
@DevinWalker  there is no issues here. But it's wrong to use lists here. Semantically speaking.

We can do the same with inline `<span>` tags.

**Comment by DevinWalker** [comment]
That's fine with me

**Comment by ramiy** [comment]
After further investigation with an accessibility expert, seems like I was wrong. It should remain a list.
